test/e2e/testutils.go

```diff
@@ -0,0 +1,132 @@
+package e2e
+
+import (
+	"context"
+	"fmt"
+	"time"
+
+	operator "github.com/operator-framework/operator-marketplace/pkg/apis/marketplace/v1alpha1"
+
+	"github.com/operator-framework/operator-sdk/pkg/test"
+	v1 "k8s.io/api/core/v1"
+	apierrors "k8s.io/apimachinery/pkg/api/errors"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/runtime"
+	"k8s.io/apimachinery/pkg/types"
+)
+
+const (
+	cleanupRetryInterval = time.Second * 1
+	cleanupTimeout       = time.Second * 30
+)
+
+// createCatalogSourceConfig returns a CatalogSourceConfig based on the provided parameters
+func createCatalogSourceConfig(catalogSourceConfigName string, namespace string, targetNamespace string, packages string) *operator.CatalogSourceConfig {
+	return &operator.CatalogSourceConfig{
+		TypeMeta: metav1.TypeMeta{
+			Kind: operator.OperatorSourceKind,
+		}, ObjectMeta: metav1.ObjectMeta{
+			Name:      catalogSourceConfigName,
+			Namespace: namespace,
+		},
+		Spec: operator.CatalogSourceConfigSpec{
+			TargetNamespace: targetNamespace,
+			Packages:        packages,
+		}}
+}
+
+// createOperatorSource returns a OperatorSource based on the provided parameters
+func createOperatorSource(operatorSourceName string, namespace string, labels map[string]string, registryNamespace string) *operator.OperatorSource {
```

Comment:
What is the value of this method? What if we wanted to create an operator source with a different name? Or an invalid namespace? IMO if there was logic in this method it would be one thing, but we could just use this syntax inline in the method and have more control without much more verbosity.

Reply:
I see your point. I will remove the methods in the interest of having more control over the objects we create for the tests.